python/valis/routes/query.py

```diff
@@ -147,3 +148,19 @@ async def carton_program(self,
 
         return list(carton_program_search(name, name_type))
 
+    @router.get('/obs', summary='Return targets with spectrum at observatory',
+                response_model=List[SDSSidStackedBase], dependencies=[Depends(get_pw_db)])
+    async def obs(self,
+                  release: Annotated[str, Query(description='Data release to query', example='IPL3')],
+                  obs: Annotated[str,
+                                 Query(enum=['APO', 'LCO'],
+                                       description='Observatory to get targets from. Either "APO" or "LCO"',
+                                       example='APO')] = 'APO',
+                  obsWave: Annotated[str,
+                                     Query(enum=['boss', 'apogee', 'all'],
```

Comment:
Can you specify default values for `obs` and `obsWave`?

Reply:
Are default values not set with how it is already written? I assumed having the `Query(...) = ...` set the default value.

Reply:
Yeah you are right!